documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,444 @@
+
+#### Creating the common mounts image 
+Run the following steps to create the common mounts image containing Model In Image model files, application archives, and the WDT installation files.
+1. Create a temporary directory for docker build context `/tmp/cm-image`.
+2. Copy the Dockefile in `/tmp/mii-sample/cm-docker-file` directory to the build context root i.e `/tmp/cm-image`.
+3. Unzip the WDT installation zip into the build context root. Remove all the `weblogic-deploy/bin/*.cmd` files which are not used in Unix environment.
+4. Create a `models` directory under the `/tmp/cm-image` directory.
+5. Copy the WDT model YAML files and properties files in `/tmp/mii-sample/model-images/model-in-image__WLS-CM-v1` directory to `/tmp/cm-image/models` directory.
+6. Stage the application archive ZIP in `/tmp/mii-sample/model-images/model-in-image__WLS-CM-v1` directory and copy the archive file to `/tmp/cm-image/models` directory.
+  - Run the following commands to create your application archive ZIP file and put it in the expected directory:
+    ```
+    # Delete existing archive.zip in case we have an old leftover version
+    ```
+    ```shell
+    $ rm -f /tmp/mii-sample/model-images/model-in-image__WLS-CM-v1/archive.zip
+    ```
+    ```
+    # Move to the directory which contains the source files for our archive
+    ```
+    ```shell
+    $ cd /tmp/mii-sample/archives/archive-v1
+    ```
+    ```
+    # Zip the archive to the location will later use when we run the WebLogic Image Tool
+    ```
+    ```shell
+    $ zip -r /tmp/mii-sample/model-images/model-in-image__WLS-CM-v1/archive.zip wlsdeploy
+    ```
+7. Copy the archive ZIP file in `/tmp/mii-sample/model-images/model-in-image__WLS-CM-v1` directory to `/tmp/cm-image/models` directory..
+8. Build the docker image by running the command `docker build --build-arg MOUNT_PATH=/common --tag model-in-image:v1 .`.
+9. Optionally, you can customize the mount path by using MOUNT_PATH build argument (it defaults to /common). 
+
+Once the image is created, it will have the WDT executables copied to `/${MOUNT_PATH}/weblogic-deploy`, and all the WDT models, variables, and archives are copied to `/${MOUNT_PATH}/models`. If you use the default mount path '/common', you can verify the contents of the image using the following commands:
+
+  ```shell
+  $ docker run -it --rm model-in-image:v1 ls -l /common
+  ```
+
+  ```shell
+  $ docker run -it --rm model-in-image:v1 ls -l /common/models
+  ```
+
+  ```shell
+  $ docker run -it --rm model-in-image:v1 ls -l /common/weblogic-deploy
+  ```
+
+#### Domain Resource
```

Comment:
This title could be more descriptive:  'Prepare and Apply the Domain Resource' or some-such...